### ISSUE TITLE
Error on mount

### ISSUE THREAD

**Issue by idonotknow** [ntt-sic/masakari-deploy#11]
It is weird when I run vagrant up,compute1 is fine with exit,but compute2 is always stuck at mount action ,error showed like this:
ERROR: mount[/mnt/instances](compute_conf::default line 30) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected process to exit with [0], but received '32'
FATAL: Chef::Exceptions::ChildConvergeError: Chef run process exited unsuccessfully (exit code 1)

**Comment by sampathP** [comment]
@idonotknow thanks for the report. Are you running the stack.sh in controller node when this happened?

**Comment by idonotknow** [comment]
@sampathP no,just when I run vagrant up,it said that chef did not successfully complete. The whole process failed before ssh to each vm.
